frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
- [ ] Please, kindly try to follow the given design as much as possible:


This is important, because on your best brand-new job, your will be required to follow a specific design.  But you will be prepared! 💪